x/validator/client/cli/tx_approve_disable_validator.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/validator/types"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdApproveDisableValidator() *cobra.Command {
+	var (
+		address string
+		info    string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "approve-disable-validator --address [address] --info [info]",
```

Comment:
Add ` [flags]` to the end of `Use` value.

`[flags]` stands for standard tx command flags like `--from` and `--yes`.